src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -69,14 +72,22 @@ function processingTableDataToApexData(
 function generateAxisLabels(
     t: TFunction,
     rowName: string,
-    colName: string
+    colName: string,
+    displayOption: DisplayOptionType
 ): [string, string] {
-    let xAxisLabel = t("Results.TableHead");
+    let xAxisLabel = t("Results.NrIsolatesText");
     let yAxisLabel = "";
     if (rowName !== "") {
-        xAxisLabel = `${t("Results.TableHead")} ${t("Results.per")} ${t(
+        xAxisLabel = `${t("Results.NrIsolatesText")} ${t("Results.per")} ${t(
             `Filters.${rowName}`
         )}`;
+        if (displayOption === "relative") {
+            xAxisLabel = `${t("Results.NrIsolatesTextRelative")} ${t("Results.per")} ${t(
+                `Filters.${rowName}`
+            )} ${t("Results.Unit")}`;
+        }
+    } else if (displayOption === "relative") {
+        xAxisLabel = `${t("Results.NrIsolatesTextRelative")} ${t("Results.Unit")}`;

```

Comment:
As the x axis label is now set differently in all if branches i suggest to reflect that in the code:
```
let xAxisLabel = "";
if(rowName !== "") {
  if(absolute) {...}
  else if(relative) {...}
} else {
  if(absolute) {...}
  else if(relative) {...}
}
```

Reply:
Done.